mayavi/tests/test_mlab_integration.py

```diff
@@ -23,6 +24,8 @@
 from mayavi.core.registry import registry
 from mayavi.tests.common import get_example_data
 
+push_exception_handler(reraise_exceptions=True)  # XXX should probably be elsewhere
```

Comment:
Why do you need this?  Where do you think this should go?  Was it helpful when running the tests?

Reply:
Ideally you could make it a suite-level fixture (`pytest`), or decorator for any file/test that uses `traits`. Otherwise you can get silent failures that only show up on `stdout` and are not actually raised.

If you remove the changes in ` mayavi/tools/modules.py`, you get a failure (TDD). If you also then uncomment this line, you just get a message on `stdout` but no failure.

Reply:
Still using nose right now, might change that at some point but for now I guess lets leave it where it is.  I'll merge it for now.  Thanks for the clarifications.